src/gui/activitywidget.h

```diff
@@ -73,9 +73,9 @@ class ActivityWidget : public QWidget
     void checkActivityTabVisibility();
 
 public slots:
-    void slotRefreshActivities(AccountStatePtr ptr);
-    void slotRefreshNotifications(AccountStatePtr ptr);
-    void slotRemoveAccount(AccountStatePtr ptr);
+    void slotRefreshActivities(const AccountStatePtr &ptr);
```

Comment:
Can we tell clazy that these requested changes make absolutely no sense? They introduce another layer of indirection which is completely unnecessary.